examples/automations.tf

```diff
@@ -0,0 +1,27 @@
+# automations.tf
+#   Automation example
+#
+# API reference:
+#   https://developer.zendesk.com/rest_api/docs/support/automations
+
+resource "zendesk_automation" "auto-close-automation" {
+  title  = "Close ticket 4 days after status is set to solved"
+  active = true
+
+  all {
+    field: "status",
+    operator: "is",
+    value: "solved"
```

Comment:
i think you can fix the build errors i saw by replacing the `:` with an `=`

Reply:
and the trailing commas should be removed, i think.